app/lib/frontend/handlers/package.dart

```diff
@@ -328,8 +336,9 @@ Future<PackagePageData> loadPackagePageData(
   final analysisView = await analyzerClient.getAnalysisView(
       selectedVersion.package, selectedVersion.version!);
 
-  final uploaderEmails = await accountBackend
-      .getEmailsOfUserIds(package.uploaders as List<String>);
+  final uploaderEmails = loadUploaderEmails
```

Comment:
Won't we need to update the documentation comment on `PackagePageData.uploaderEmails` to indicate that this being `null` doesn't necessarily mean that it has a publisher, because it might have been loaded with `loadUploaderEmails`.

See earlier comment about maybe removing `PackagePageData.uploaderEmails` and loading it in `packageAdminHandler` instead.. since that's the only place we use it.